packages/scene-composer/src/store/helpers/serializationHelpers.ts

```diff
@@ -127,19 +127,14 @@ function createTagComponent(
 
   validateRuleBasedMapId(ruleBasedMapId, resolver, errorCollector);
 
-  let dataBindingContext: any;
-  if (valueDataBinding) {
-    dataBindingContext = valueDataBinding.dataBindingContext;
-  }
-
   return {
     ref: generateUUID(),
     type: 'Tag',
     icon,
     chosenColor: component.chosenColor,
     ruleBasedMapId,
     valueDataBinding: {
-      dataBindingContext,
+      ...(valueDataBinding || {}),
```

Comment:
Linting configs, such as TypeScript ESLint, are pushing for using nullish coalescing (`??`) over short-circuiting. What you're doing is valid and fine, but it's becoming less preferred.

Reply:
good to know. I will use `??` next time